src/_vars.scss

```diff
@@ -54,3 +58,41 @@ $break-xl: 1536px;
 $header-height-xs: 56px;
 $header-height-sm: 64px;
 $header-height-md: 72px;
+
+// TEMPORARY to hold original values
+// $primary-blue: #014983;
+// $primary-cyan: #00aeef;
+// $secondary-green: #b2bb1c;
+// $secondary-yellow: #fdb913;
+// $secondary-orange: #de571f;
+// $secondary-red: #b53228;
+
+// $alternative-red: #ff0000;
+
+// $neutral-white: #ffffff;
+// $neutral-white-opacity80: #ffffffcd;
+// $neutral-white-opacity50: #ffffff80;
+// $neutral-white-opacity30: #ffffff4d;
+// $neutral-light-gray: #ebeaea;
+// $neutral-light-gray2: #dddddd;
+// $neutral-gray: #808080;
+// $neutral-gray2: #cccccb;
+// $neutral-gray3: #adadad;
+// $neutral-dark-gray: #31342b;
+// $neutral-dark-gray2: #444444;
+// $neutral-dark-gray3: #555555;
+
+// $neutral-black: #000000;
+// $neutral-black-opacity90: #000000e6;
+// $neutral-black-opacity55: #0000008c;
+// $neutral-black-opacity40: #00000066;
+// $neutral-black-opacity20: #00000033;
+// $neutral-black-opacity15: #00000026;
+// $neutral-black-opacity125: #00000020;
+// $neutral-black-opacity10: #0000001a;
+
+// $dark-text-color: rgba(0, 0, 0, 0.87);
+
+// $wellness-red: #a30000;
+// $wellness-green: #005500;
+// $wellness-yellow: #ffcc00;
```

Comment:
This seems unnecessary, since you still have the original values above.

Reply:
Yeah, sorry, meant to get rid of that

Reply:
Why is it still here?